bench/db_bench.ml

```diff
@@ -35,7 +17,7 @@ let hash_size = 30
 
 let value_size = 13
 
-let nb_entries = 10_000_000
+let nb_entries = 10_000
```

Comment:
is that change necessary? Maybe we want that number to be passed on the command line to ease tweaking?

Reply:
Whoops, this should not be changed. Also this is from #134 (I'll pull in that branch once it gets merged)